src/.nuget/packages.OSX.config

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ValMenn 

We run Nuget Install on the dependencies under /src/.nuget to get some of the required bootstrapping tools. My current understanding is that Nuget Install works with the xml format.